src/magick-image.ts

```diff
@@ -104,6 +104,11 @@ export interface IMagickImage extends INativeInstance {
     compositeGravity(image: IMagickImage, gravity: Gravity, compose: CompositeOperator, point: Point, channels: Channels): void;
     compositeGravity(image: IMagickImage, gravity: Gravity, compose: CompositeOperator, point: Point, args: string): void;
     compositeGravity(image: IMagickImage, gravity: Gravity, compose: CompositeOperator, point: Point, args: string, channels: Channels): void;
+    contrast(): void;
+    contrast(enhance: boolean): void;
```

Comment:
I think I rather have an overload instead of a boolean argument. Not sure what this should be called though. Only name I can come up with now is `contrastWithoutEnhance`. Do you have any better ideas?

Reply:
Perhaps `decontrast`, `discontrast`, `anticontrast`, or `decreaseContrast`?

Reply:
Thanks for those ideas and that just gave me another idea. Maybe `inverseContrast` to be more in line with the other `inverse` methods?